Lib/test/test_types.py

```diff
@@ -1819,6 +1819,14 @@ class Model(metaclass=ModelBase):
         with self.assertRaises(RuntimeWarning):
             type("SouthPonies", (Model,), {})
 
+    def test_tuple_subclass_as_bases(self):
+        # gh-132176: it used to crash on using
+        # tuple subclass for as base classes.
+        class TupleSubclass(tuple): pass
+
+        typ = type("typ", TupleSubclass((int, object)), {})
+        self.assertEqual(typ.__bases__, (int, object))
```

Comment:
I suggest adding the test: `self.assertEqual(type(typ.__bases__), TupleSubclass)`.

I expected the `tuple` type, but I'm fine with `TupleSubclass` :-)